CefSharp.BrowserSubprocess.Core/Async/JavascriptAsyncMethodHandler.h

```diff
@@ -0,0 +1,38 @@
+// Copyright � 2010-2015 The CefSharp Project. All rights reserved.
+//
+// Use of this source code is governed by a BSD-style license that can be found in the LICENSE file.
+
+#pragma once
+
+#include "include/cef_v8.h"
+#include "JavascriptCallbackRegistry.h"
+#include "JavascriptAsyncMethodCallback.h"
+
+namespace CefSharp
+{
+    namespace Internals
+    {
+        namespace Async
+        {
+            class JavascriptAsyncMethodHandler : public virtual CefV8Handler
+            {
+            private:
+                gcroot<JavascriptCallbackRegistry^> _callbackRegistry;
+                gcroot<Func<JavascriptAsyncMethodCallback^, int64>^> _methodCallbackSave;
+                CefRefPtr<CefV8Value> _promiseCreator;
+                int64 _objectId;
+
+            public:
+                JavascriptAsyncMethodHandler(int64 objectId, JavascriptCallbackRegistry^ callbackRegistry, CefRefPtr<CefV8Value> promiseCreator, Func<JavascriptAsyncMethodCallback^, int64>^ methodCallbackSave)
```

Comment:
I don't think it's necessary, both the gcroots and CefRefPtr will take care of it in their destructors. and callback registry shouldn't be disposed here.